### PR TITLE
allow redirect of gateway root to console URL

### PR DESCRIPTION
**Pull request by apoland**
interested in comments.. will this break anything the gateway is currently doing?

### CODE REVIEW

File: service.go [hunk 1]
```diff
@@ -55,6 +55,7 @@ type API struct {
 		ReadHeaderTimeout time.Duration        `json:"read_header_timeout"`
 		WriteTimeout      time.Duration        `json:"write_timeout"`
 		IdleTimeout       time.Duration        `json:"idle_timeout"`
+		UIRedirect        string               `json:"ui_redirect"`
```
Comment (ronenh) [inline service.go]: I'd rename the field to `RootRedirect` to clarify what it's about.

Comment (gertd) [inline service.go]: Does this need to be config based, aka another configuration value and change, when the values seems hardwired?

Comment (apoland) [reply service.go]: Seems like topaz configures service-host so topaz should tell service-host what the redirect URL is?  But I'm ok with hard coding it here if we want to.

Comment (apoland) [reply service.go]: Can topaz pass a default so this is not required to be in the config file?

File: service_factory.go [hunk 2]
```diff
@@ -110,7 +110,13 @@ func (f *ServiceFactory) prepareGateway(config *API, gatewayOpts *GatewayOptions
 	}
 
 	mux := http.NewServeMux()
-	mux.Handle("/", runtimeMux)
+	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
```
Comment (gertd) [inline service_factory.go]: This method is generic for all gateway instance, I think the console specific change could be made in https://github.com/aserto-dev/topaz/blob/1bd29a25990c1963314032f4c5d61c70ac4e6837/pkg/app/console.go#L34C26-L34C48

**Comment by BogdanIrimie** [comment]
@apoland after talks with @carabasdaniel I opened https://github.com/aserto-dev/topaz/pull/329 that should not require any changes in service-host. If that PR seams ok, we can close this one.

**Pull request closed without merging**